Source/WebCore/platform/graphics/displaylists/DisplayListItem.h

```diff
@@ -208,14 +208,18 @@ struct ApplyItemResult {
     std::optional<RenderingResourceIdentifier> resourceIdentifier;
 };
 
+enum class RecordReplayOption : uint8_t {
```

Comment:
How about `ReplayOption`? The full name is `DisplayList::ReplayOption`

Reply:
Hmm, I thought about that, but it indicates the option would only affect replay, but it also affects recording.